components/eventing-controller/pkg/backend/nats/jetstream/jetstream.go

```diff
@@ -29,12 +29,15 @@ import (
 var _ Backend = &JetStream{}
 
 const (
-	jsHandlerName          = "jetstream-handler"
-	idleHeartBeatDuration  = 1 * time.Minute
-	jsConsumerMaxRedeliver = 100
-	jsConsumerAcKWait      = 30 * time.Second
-	jsMaxStreamNameLength  = 32
-	separator              = "/"
+	jsHandlerName             = "jetstream-handler"
+	idleHeartBeatDuration     = 1 * time.Minute
+	jsConsumerMaxRedeliver    = 100
+	jsConsumerAcKWait         = 30 * time.Second
+	jsMaxStreamNameLength     = 32
+	separator                 = "/"
+	NoNatsSubscriptionErr     = "failed to create NATS JetStream subscription"
```

Comment:
why not add `for subject: %v` here itself?

Reply:
The problem is that I need a generic error message for [this condition](https://github.com/kyma-project/kyma/pull/15447#discussion_r969600253).